lib/index.js

```diff
@@ -19,9 +20,10 @@ module.exports = curry['default'] = curry
 function curry (fn) {
   var self = this
   return function () {
-    var args1 = slice(arguments)
+    var args = slice(arguments)
```

Comment:
Also, @ashaffer would like to move away from using slice for performance reasons... and inline args creations..

```js
var args = []
for (var i = 0; i < arguments.length; i++) {
  args.push(arguments[i])
}
```